src/coffee/import.coffee

```diff
@@ -416,6 +418,8 @@ class Import
 
   update: (product, existingProduct, id2SameForAllAttributes, header, rowIndex, publish) ->
     product.categoryOrderHints = @_mergeCategoryOrderHints existingProduct, product
+    if (not product.state? and not existingProduct.state? and @defaultState?)
```

Comment:
Just a nitpick: in this case, you don't have to add a question mark - for example here: `product.state?` it checks that the state is not `undefined`. If yes, it would return `false`. Without the question mark, it would return `undefined` which is also a `false value`. 

On the other hand, if the product can be `undefined`, it would make sense to write: `product?.state` which would ensure that you don't try to access a property state of an undefined.

But this will also work and I am more than ok with leaving it here - just to be sure.